pkg/k8s/apis/cilium.io/v2alpha1/bgp_peer_types.go

```diff
@@ -105,6 +105,11 @@ type CiliumBGPFamily struct {
 	// +kubebuilder:validation:Enum=unicast;multicast;mpls_label;encapsulation;vpls;evpn;ls;sr_policy;mup;mpls_vpn;mpls_vpn_multicast;route_target_constraints;flowspec_unicast;flowspec_vpn;key_value
 	// +kubebuilder:validation:Required
 	Safi string `json:"safi"`
+}
+
+// CiliumBGPFamilyWithAdverts represents a AFI/SAFI address family pair along with reference to BGP Advertisements.
+type CiliumBGPFamilyWithAdverts struct {
+	CiliumBGPFamily `json:",inline"`
```

Comment:
Why is CiliumBGPFamilyWithAdverts needed when it has the same fields as CiliumBGPFamily?

Reply:
CiliumBGPFamily will not contain Advertisements after this change.